<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTypeFrameWidget</class>
 <widget class="QWidget" name="ViewTypeFrameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="rbtnOnlyVideo">
       <property name="text">
        <string>only video</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroupViewTypes</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtnAnnotationAndVideo">
       <property name="text">
        <string>annotation and Video</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroupViewTypes</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtnAnnotationWithoutVideo">
       <property name="text">
        <string>annotation without video</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">rbtnGroupViewTypes</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="rbtnGroupViewTypes"/>
 </buttongroups>
</ui>
